worker/caasfirewallersidecar/applicationworker.go

```diff
@@ -165,6 +156,20 @@ func (w *applicationWorker) loop() (err error) {
 			if !ok {
 				return errors.New("application watcher closed")
 			}
+
+			// If charm is (now) a v1 charm, exit the worker.
```

Comment:
A charm can not transition from v2 to v1.
We should never start this worker for a v1 charm, so this charm format check is irrelevant I suspect and is not needed.

Reply:
Good call, removed "(now)"